packages/attestation-service/src/sms/nexmo.ts

```diff
@@ -70,6 +71,15 @@ export class NexmoSmsProvider extends SmsProvider {
       phoneNumber: number.msisdn,
       code: phoneUtil.getRegionCodeForNumber(phoneUtil.parse('+' + number.msisdn)),
     }))
+
+    // The US number is likely a toll-free number and cannot be used outside of US and Canada
+    // We therefore want to set the default number to be non-US for global reach
```

Comment:
This assumes the operator deletes any existing US numbers, otherwise `getMatchingNumber` code below could be mapped to that other US number. This means we'd need to suggest an alternative international number to send non-US SMS.

We've previously instructed people to use a US number as their default number (and in many cases it is their only number). I don't feel we know enough about whether a sender from X country could be as successful globally. WDYT about adding logic instead to set the default to the first non-tfn number (so not changing their default effectively) and alter the `getMatchingNumber` function to have a special usecase for US -- filter for all US numbers and prefer the TFN if it exists.

it actually looks like the vonage number insights API returns tollfree as a line type (though not sure if this requests costs $$)
https://developer.nexmo.com/number-insight/code-snippets/number-insight-standard

> The line type (mobile/landline/virtual number/premium/toll-free)

Alternatively, we could just filter by the number codes themselves, should be standardized to 800, 888, 877, 866, 855, 844 or 833.
https://www.fcc.gov/consumers/guides/what-toll-free-number-and-how-does-it-work

Reply:
or maybe there is an existing library that can take the number and check for the tfn prefix

Reply:
Great point Amy! This approach will be less confusing to AS operators.